packages/commons/package.json

```diff
@@ -44,6 +48,7 @@
     "@types/node": "20.12.12",
     "@types/uuid": "9.0.8",
     "eslint": "8.57.0",
+    "cpx2": "2.0.0",
```

Comment:
Also this one was just updated on main 🥲 https://github.com/pagopa/interop-be-monorepo/pull/533